src/ExternalSearch.Providers.DuckDuckgo/DuckDuckGoExternalSearchProvider.cs

```diff
@@ -429,7 +429,10 @@ private static T JsonRequestWrapper<T>(IRestClient client, string parameter, Met
         /// <param name="label">The label to format</param>
         /// <returns>The formatted label</returns>
         private static string FormatLabelToProperty(string label)
-        {
+        {
+            if (string.IsNullOrEmpty(label))
+                return string.Empty;

```

Comment:
should return null as the caller checks for null. If we return empty it will try to create invalid keys

Reply:
Perhaps IsNullOrWhitespace might be better

Reply:
Done